ios/Classes/FacebookModule.m

```diff
@@ -574,17 +580,25 @@ -(void)presentShareDialog:(id)args
     TiThreadPerformOnMainThread(^{
         FBSDKShareLinkContent *content = [[FBSDKShareLinkContent alloc] init];
         content.contentURL = [NSURL URLWithString:[params objectForKey:@"link"]];
-        content.contentDescription = [params objectForKey:@"description"];
-        if ([params objectForKey:@"name"] != nil) {
-            DEPRECATED_REPLACED_REMOVED(@"Facebook.presentShareDialog.name", @"5.0.0", @"5.0.0", @"Titanium.Facebook.presentShareDialog.title");
+        
+        NSString *description = [params objectForKey:@"description"];
+        NSString *title = [params objectForKey:@"title"];
+        NSString *picture = [params objectForKey:@"picture"];
+        
+        if (description != nil) {
+            NSLog(@"Ti.Facebook.presentShareDialog.description has been deprecated in Ti.Facebook 5.5.0 as of the Graph v2.9 changes.");
+            content.contentDescription = [params objectForKey:@"description"];
```

Comment:
As per FBSDKShareLinkContent.h -
@property (nonatomic, readonly) NSString *contentDescription
DEPRECATED_MSG_ATTRIBUTE("contentDescription is deprecated from Graph API 2.9");

contentDescription has changed to read only, but you are assigning to it. It is giving build error. Please remove it.